### PR TITLE
fix sample hostendpoint policies in bare-metal docs for v3

### PR DESCRIPTION
**Pull request by 2ffs2nns**
## Description
fix sample hostendpoint policies in bare-metal docs for v3

## Release Note
```release-note
None required
```

### CODE REVIEW

File: master/getting-started/bare-metal/bare-metal.md [hunk 1]
```diff
@@ -318,11 +318,11 @@ cat << EOF | calicoctl create -f -
   kind: HostEndpoint
   metadata:
     name: <name of endpoint>
-    node: <node name or hostname>
     labels:
```
Comment (bcreane) [inline master/getting-started/bare-metal/bare-metal.md]: I don't see `labels` anymore in the struct. Did you try that out?

Comment (2ffs2nns) [reply master/getting-started/bare-metal/bare-metal.md]: works fine w/sample yaml and can be queried...
```
./calicoctl get hostEndpoint --out=go-template="{{range .}}{{range .Items}}{{.ObjectMeta.Labels}},{{end}}{{end}}"
map[role:webserver environment:production],map[host-endpoint:ingress],map[type:production],
```

Comment (bcreane) [reply master/getting-started/bare-metal/bare-metal.md]: never mind, that's in `metav1.ObjectMeta`

**Comment by emanic** [comment]
LGTM thanks @2ffs2nns 👍

**Pull request merged**